editor/src/components/canvas/controls/outline-control.tsx

```diff
@@ -124,10 +130,72 @@ export const OutlineControls = (props: OutlineControlsProps) => {
     [props.canvasOffset.x, props.canvasOffset.y, props.componentMetadata, props.dragState],
   )
 
-  const anySelectedElementIsYogaLayouted = anyInstanceYogaLayouted(
-    props.componentMetadata,
-    props.selectedViews,
-  )
+  let parentHighlights: React.ReactNode = null
+  if (props.keysPressed['cmd'] || layoutInspectorSectionHovered) {
+    parentHighlights = props.selectedViews.map((view) => {
```

Comment:
This should probably be wrapped in a `React.useMemo`

Reply:
Fixed in 8b4f17d8.